package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/applicationinsights-angularplugin-js",
-  "version": "2.6.4",
+  "version": "2.6.5",
```

Comment:
suggestion: Lets bump this to 2.7 to match the AI v2.7 -- just to make it easier to track dependent versions 😄